src/torchio/transforms/augmentation/spatial/random_flip.py

```diff
@@ -85,51 +84,31 @@ class Flip(SpatialTransform):
         image orientation is not known.
     """
 
-    def __init__(self, axes, **kwargs):
+    def __init__(
+        self, axes: Union[int, Tuple[int, ...], str, Tuple[str, ...]], **kwargs
+    ):
         super().__init__(**kwargs)
-        self.axes = _parse_axes(axes)
-        self.args_names = ('axes',)
```

Comment:
I suspect the type checker is checking that this is a tuple somewhere, but I might be wrong.